packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -63,7 +73,34 @@ class Chat extends Widget<Properties> {
   _renderMessageBox(): void {
     const $messageBox = $('<div>').appendTo(this.element());
 
-    this._messageBox = this._createComponent($messageBox, MessageBox, {});
+    const configuration: MessageBoxProperties = {
+      onSendButtonClick: (text) => {
```

Comment:
I've corrected the names. But you seem to be confused. What you are asking to fix is the internal API of the MessageBox private component, not the chat API. It doesn't need to be coordinated with the technical writers

Reply:
I understand that this is an internal component, and at the moment, I have suggested two options: onSendMessage and onMessageSend. In our public component, `onMessageSend` is used
https://github.com/marker-dao/DevExtreme/blob/24_2_chat_on_message_send_api/packages/devextreme/js/ui/chat.d.ts#L126-L127

I don't know if that name was agreed upon because, for me, onSendMessage seems more logical. That's why I brought it to your attention.